applications/MeshingApplication/custom_utilities/uniform_refine_utility.cpp

```diff
@@ -209,10 +209,11 @@ void UniformRefineUtility<TDim>::RefineLevel(const int& rThisLevel)
         {
             // Loop the edges of the father element and get the nodes
             int i_edge = 0;
-            std::array<NodeType::Pointer, 5> middle_nodes;
+            std::vector<NodeType::Pointer> middle_nodes(5);
```

Comment:
Idem

Reply:
I have used vector to avoid code duplication. Another method uses `middle_nodes` as input parameter (to add the nodes to the sub model parts). This method takes the `middle_nodes` as a variable sized vector (3 or 5)

Reply:
Ok, feel free to merge